public-ui/src/app/data-browser/concept-charts/concept-charts.component.ts

```diff
@@ -112,4 +125,10 @@ export class ConceptChartsComponent implements OnInit, OnDestroy {
       analysis.results.push(this.otherGenderResult);
     }
   }
+
+    showHistogram(unit: string) {
```

Comment:
This code in this function deals with just the gender analysis? Can you make a better name for it. And reuse it above where  this code is duplicated?

Reply:
Done.